wellregistry/registry/admin/fetch_from_nwis.py

```diff
@@ -44,7 +45,15 @@ def _validate_site(site_data):
         return True, 'Valid site'
 
     @staticmethod
-    def _get_monitoring_location(site_data, user):
+    def _get_lcl_aqfr_name(dictnry, aqfr_cd, state_cd):
+        for item in list(dictnry):
+            if item['Aqfr_Cd'] == aqfr_cd and item['State_Cd'] == state_cd:
+                print(item['Aqfr_Nm'] + " for " +  aqfr_cd)
+                return item['Aqfr_Nm']
+        return {'status' : "key not found"}
+
+    @staticmethod
+    def _get_monitoring_location(self, site_data, user):
```

Comment:
Please remove self. Marking this as a staticmethod means we do not need self. The Codacy check also flagged this.

Reply:
resolved